cluster/gce/container-linux/configure-helper.sh

```diff
@@ -34,6 +34,22 @@ function create-dirs {
   fi
 }
 
+# Vars assumed:
```

Comment:
I was thinking about this today as well. I can see some cases where it would be nice to have some common.sh for the duplicate logic, however, I'm thinking for now lets not worry about that in this PR.  I think maybe that can be handled in a separate PR if we really feel like this is painful to have the dup logic and there are probably multiple places that would benefit from this.